.github/workflows/ci.yml

```diff
@@ -126,13 +126,21 @@ jobs:
                 else
                     sed -i "/mpi4py/ d" requirements.txt
                 fi
+
                 build_py_project_in_conda_env
+                test_py_project
+                cd ..
 
                 if test "$DOWNSTREAM_PROJECT" = "mirgecom"; then
                     # Test main branch
                     examples/run_examples.sh ./examples
+
+                    # Test production drivers
```

Comment:
Maybe this should be a separate CI job? mirgecom is already the longest job in grudge, I'm reluctant to make CI turnaround even more sluggish.

Reply:
Then maybe drop the downstream `pytest` (which takes a long time)?  Running the examples (typically done within 10 minutes) will be enough and much faster.  The examples won't work if *MIRGE-Com* is broken, and they will also fail if they get anything outside the accepted solution range.